### PR TITLE
feat(DW): gray donuts for empty data top-consuming charts

### PR DESCRIPTION
**Pull request by gitdallas**
addresses feedback on empty charts in the "top-consuming" section while we have data in other charts (happens when winding down?). also adds empty state for status overview chart

charts with no data will now show a gray donut, with a "No workload is consuming {unit}" tooltip and no legend.

## Description
charts with no data will now show a gray donut, with a "No workload is consuming {unit}" tooltip and no legend.
image
image

## How Has This Been Tested?
manually modifying data to test with empty data

## Test Impact
no test impact, mostly a design change that doesn't impact data

## Request review criteria:
no workloads at all will still show the empty state screen, if there is data a specific chart doesn't have data, it will get the new gray donut

Self checklist (all need to be checked):
- [x] The developer has manually tested the changes and verified that the changes work
- [x] Commits have been squashed into descriptive, self-contained units of work (e.g. 'WIP' and 'Implements feedback' style messages have been removed)
- [x] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [x] The developer has added tests or explained why testing cannot be added (unit or cypress tests for related changes)

If you have UI changes: 

- [x] Included any necessary screenshots or gifs if it was a UI change.
- [ ] Included tags to the UX team if it was a UI/UX change (find relevant UX in the [SMEs](https://github.com/opendatahub-io/odh-dashboard/tree/main/docs/smes.md) section).

After the PR is posted & before it merges:
- [ ] The developer has tested their solution on a cluster by using the image produced by the PR to `main`

### CODE REVIEW

**Review by lucferbux — APPROVED** [review]
/lgtm

File: frontend/src/pages/distributedWorkloads/global/workloadStatus/DWStatusOverviewDonutChart.tsx [hunk 1]
```diff
@@ -32,6 +33,11 @@ export const DWStatusOverviewDonutChart: React.FC = () => {
     (statusType) => statusCounts[statusType] > 0,
   );
 
+  if (workloads.data.length) {
```
Comment (mturley) [inline frontend/src/pages/distributedWorkloads/global/workloadStatus/DWStatusOverviewDonutChart.tsx:36]: This should be `!workloads.data.length`, missing the `!`

**Review by mturley — CHANGES_REQUESTED** [review]

**Review by mturley — APPROVED** [review]

**Comment by lucferbux** [comment]
/override ci/prow/images ci/prow/odh-dashboard-pr-image-mirror

**Pull request merged**